src/features/registry.js

```diff
@@ -58,4 +58,25 @@ systemJSPrototype.delete = function (id) {
         depLoad.i.splice(importerIndex, 1);
     });
   return delete this[REGISTRY][id];
-};
\ No newline at end of file
+};
+
+systemJSPrototype.entries = function () {
+  const registry = this[REGISTRY];
+  return Object.keys(registry).map(function (key) {
+    return [key, registry[key].n];
+  });
+}
```

Comment:
I was thinking more like the `Map.prototype.entries()` approach which is just the same as the iterator.